kong/consumer_group_test.go

```diff
@@ -55,6 +55,36 @@ func TestConsumerGroupsService(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestConsumerGroupWithTags(t *testing.T) {
+	RunWhenEnterprise(t, ">=3.1.1", RequiredFeatures{})
+	require := require.New(t)
+	assert := assert.New(t)
+
+	client, err := NewTestClient(nil, nil)
+	require.NoError(err)
+	require.NotNil(client)
+
+	cg := &ConsumerGroup{
+		Name: String("foo"),
+		Tags: StringSlice("tag1", "tag2"),
+	}
+
+	createdConsumerGroup, err := client.ConsumerGroups.Create(defaultCtx, cg)
+	require.NoError(err)
+	t.Cleanup(func() {
+		err = client.ConsumerGroups.Delete(defaultCtx, createdConsumerGroup.ID)
+		require.NoError(err)
```

Comment:
```suggestion
		assert.NoError(err)
```

I realize that there's no other deferred functions or cleanups but better to use `assert` here just in case there's something added to run on test teardown to not prevent the further cleanup being done.